### PR TITLE
Make changes from code reviewer

### PR DESCRIPTION
**Pull request by Kajal19-del**
# In this milestone I did the following activities:
- I setup webpack first
- I add HTML, CSS, and javascript files for the to-do-list project
- I set up a module folder for different JS file
- I add (Add & Remove) task features for this project
- I pass  all the linter errors
__here is the link to the previous PR https://github.com/Kajal19-del/To-do-list/pull/2__

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @Kajal19-del , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project for re-review, but you are not submitting the same PR that you have submitted for the first time.
we have to submit the same PR for the re-reviews.
Also this PR only have one file so I cant review it properly using Github features like inline-comments.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kajal19-del** [comment]
The previous pull request was merged by mistake but I put the old PR link below
https://github.com/Kajal19-del/To-do-list/pull/2

File: src/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		  
 		  <section id="todoList"></section>
 		  
-		  <div class="clearAll">Clear all completed</div>
+		  <button class="clearAll">Clear all completed</button>
```
Comment (Tufahel) [inline src/index.html:28]: - [ ] You have implemented `Clear All Completed` function in `index.js`. This function is for deleting all selected lists. Now as per the requirement you need to implement another function in `index.js` and button to delete a specific list.

- ***Requirement***
image

- ***Suggestion on your design***
image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Kajal19-del   :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.


## Required Changes ♻️

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Kajal19-del 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**